platform/fio/api.go

```diff
@@ -22,3 +22,20 @@ func (p *Platform) Coin() coin.Coin {
 func (p *Platform) GetTxsByAddress(address string) (page blockatlas.TxPage, err error) {
 	return page, err
 }
+
+func (p *Platform) Lookup(coins []uint64, name string) ([]blockatlas.Resolved, error) {
+	var result []blockatlas.Resolved
+	for _, coinId := range coins {
+		coinObj := coin.Coins[uint(coinId)]
+		address, err := p.client.lookupPubAddress(name, coinObj.Symbol)
+		if err != nil {
```

Comment:
Maybe we can ignore coins with errors and return the correct ones
If I will have an array of coins where only 1 is incorrect - it is better to return all the info except the invalid coin?

Reply:
Good suggestion.  This is a matter of taste/usage pattern.  After some consideration I think in partial error case the caller is more likely to handle error if error is returned than if partial error is returned.  That has the risk of not noting error, and attempting to use missing address.  Better to return error if not all could be resolved.  Typical use case is with one coin anyways.